Rspec/spec/author_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../../author'
+require_relative '../../item'
+require 'rspec'
+
+describe Author do
+  let(:first_name) { 'Kevin' }
+  let(:last_name) { 'Toro' }
+  let(:author) { Author.new(first_name, last_name) }
+
+  describe '#initialize' do
+    it 'should receive and set first and last names of author' do
+      expect(author.first_name).to eql(first_name)
+      expect(author.last_name).to eql(last_name)
+    end
+    it 'first and last names should be strings' do
+      expect(author.first_name).to be_a(String)
+      expect(author.last_name).to be_a(String)
+    end
+  end
```

Comment:
When I run 'Rspec filename_rspec.rb' I should see the tests running with examples and tests passed, I think you need to fix that to ensure that your unit testing is complete

Reply:
Hello @torobucci 
It gives this error "No examples found" despite I installed the required packages after switching to the branch.
![noexamples](https://github.com/torobucci/catalog-of-my-things/assets/26763433/88a6fa9d-593f-4241-8215-fa710494ac13)